### PR TITLE
Changes the thumbs up and down reacts to horse and fish reacts

### PR DESCRIPTION
**Pull request by LoliconSlayer**

### CODE REVIEW

**Comment by Burger1998** [comment]
SPEEDMERGE NOW

**Comment by Burger1998** [comment]
WHERES THE MERGE PEOPLE WE SHOULD MERGED THIS THE HOUR IT WAS MADE

File: src/main/java/net/yogstation/yogbot/listeners/channel/MemesChannel.kt [hunk 1]
```diff
@@ -20,9 +20,9 @@ class MemesChannel(channelsConfig: DiscordChannelsConfig) : AbstractChannel(chan
 			message.embeds.size > 0 ||
 			memetypes.contains(message.content.split(".").last())
 			) {
-			// Java is strange with unicode in strings, this is thumbs up and down emoji
-			return message.addReaction(ReactionEmoji.unicode("\uD83D\uDC4D"))
-				.and(message.addReaction(ReactionEmoji.unicode("\uD83D\uDC4E")))
+			// Java is strange with unicode in strings, this is horse and fish emoji
```
Comment (AshCorr) [inline src/main/java/net/yogstation/yogbot/listeners/channel/MemesChannel.kt:23]:
```suggestion
			// Java is strange with unicode in strings, this is 🐎 and 🐟emoji
```

Emojis are allowed in Java :)

**Comment by AshCorr** [comment]
@JamieD1 BOOOOO

**Comment by moonheart08** [comment]
i'm here to post an among us before this gets locked. (you should probably lock this)
image

**Comment by Bibby0110** [comment]
24 hour rule

**Comment by ReddicusDragon** [comment]
FUCK OFF BIBBY, IT WAS DONE IN 2022

**Pull request closed without merging**